setup.py

```diff
@@ -55,6 +55,9 @@
         "fonttools>=3.24.0",
         "defcon>=0.3.0",
     ],
+    extras_require = {
+        "ufo_normalization":  ["ufonormalizer"]
```

Comment:
yeah I thought about that, then I thought... hm, the name of the extra is longer than the name of the only extra requirement, "ufonormalizer" :D
But it's ok, leave it in. I doubt anyone will ever use it, but at least it's an explicit signal that there is an optional requirement.